app/javascript/pages/account/create-account.tsx

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable @typescript-eslint/unbound-method */
+import React from "react"
+
+import { AppearanceStyleType, Button, Form, t } from "@bloom-housing/ui-components"
+import { Card } from "@bloom-housing/ui-seeds"
+
+import withAppSetup from "../../layouts/withAppSetup"
+import Layout from "../../layouts/Layout"
+import NameFieldset from "./components/NameFieldset"
+import { useForm } from "react-hook-form"
+import DOBFieldset from "./components/DOBFieldset"
+import EmailFieldset from "./components/EmailFieldset"
+import PasswordFieldset from "./components/PasswordFieldset"
+import { FormHeader, FormSection } from "../../util/accountUtil"
+import "./styles/account.scss"
+
+interface CreateAccountProps {
+  assetPaths: unknown
+}
+
+const CreateAccountFormSection = ({ children }: { children: React.ReactNode }) => {
+  return <FormSection className="py-4 md:py-8 md:px-10">{children}</FormSection>
+}
+
+const NameSection = () => {
+  const {
+    register,
+    formState: { errors },
+  } = useForm({ mode: "all" })
+
+  return (
+    <CreateAccountFormSection>
+      <NameFieldset
+        register={register}
+        errors={errors}
+        defaultFirstName={null}
+        defaultMiddleName={null}
+        defaultLastName={null}
+      />
```

Comment:
Would if be better to make these default fields optional? That way we aren't having to pass `null`

Reply:
oooh yeah